### PR TITLE
Fixture/update config files

### PR DESCRIPTION
**Pull request by steven-heng**
Npm audit found 0 vulnerabilities
Modify rules command of package.json and Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +24,6 @@ ifdef CI_PULL_REQUEST
 	@sonar-scanner-2.8/bin/sonar-scanner -e -Dsonar.analysis.mode=preview -Dsonar.github.pullRequest=${shell basename $(CI_PULL_REQUEST)} -Dsonar.github.repository=$(REPO_SLUG) -Dsonar.github.oauth=$(GITHUB_TOKEN) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS) -Dsonar.host.url=$(SONAR_HOST_URL)
 endif
 ifeq ($(CIRCLE_BRANCH),develop)
-	@sonar-scanner-2.8/bin/sonar-scanner -e -Dsonar.analysis.mode=publish -Dsonar.host.url=$(SONAR_HOST_URL) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS)
+	@sonar-scanner-2.8/bin/sonar-runner -e -Dsonar.analysis.mode=publish -Dsonar.host.url=$(SONAR_HOST_URL) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS)
```
Comment (jkernech) [inline Makefile]: Revert your changes here, `sonar-runner` is deprecated so you need to use `sonar-scanner`

**Review by jkernech — CHANGES_REQUESTED** [review]

**Review by jkernech — APPROVED** [review]

**Review by jkernech — CHANGES_REQUESTED** [review]
Tests have to be fixed

File: src/modules/message.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-const tracer = require('./tracer.js');
 const serializeError = require('serialize-error');
 const CircularJSON = require('circular-json');
+const tracer = require('./tracer.js');
 
 function jsonify(obj) {
```
Comment (dialonce) [inline src/modules/message.js:5]: CRITICAL Refactor this function to reduce its Cognitive Complexity from 16 to the 15 allowed. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)
MAJOR Refactor this function to always return the same type. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3800)

File: src/modules/message.js [hunk 3]
```diff
@@ -123,7 +123,7 @@ class Message {
     @param settings {Object} - stream settings. Falls back to {} if not given
     @param delimiter {string} - a character to split prefix parts. Falls back to ':'
     @return {object} - Prefix for the log message. Or an empty string of no prefix data logging is enabled
-  **/
+   */
   getPrefix(settings = {}, delimiter = ':') {
```
Comment (dialonce) [inline src/modules/message.js:127]: CRITICAL Refactor this function to reduce its Cognitive Complexity from 18 to the 15 allowed. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)

File: src/subscribers/console-link.js [hunk 4]
```diff
@@ -29,35 +30,35 @@ class ConsoleLink extends Subscriber {
...
-    @param message {Object} - message package object
-    @see LoggerStream message package object structure description
-  **/
+   * @function handle
+   * Process a message and log it if the subscriber is switched on and message's log level is >= than MIN_LOG_LEVEL
+   * Finally, pass the message to the next subscriber if any
+   * @param message {Object} - message package object
+   * @see LoggerStream message package object structure description
+   */
   handle(message) {
```
Comment (dialonce) [inline src/subscribers/console-link.js:62]: CRITICAL Refactor this function to reduce its Cognitive Complexity from 19 to the 15 allowed. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)

**Comment by dialonce** [comment]
SonarQube analysis reported 5 issues
* ![CRITICAL][CRITICAL] 4 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [index.js#L175](https://github.com/dial-once/node-logtify/blob/b2e4446a5e3296039be740979be314ebb95114b8/src/index.js#L175): Refactor this function to reduce its Cognitive Complexity from 16 to the 15 allowed. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**